test/unit/ObjectMakrTests/make.js

```diff
@@ -0,0 +1,38 @@
+define(["mocks"], function (mocks) {
+    return function () {
+        var expect = require("chai").expect;
+
+        it("object's name is John the Penguin", function () {
+            // Arrange
+            var objectMaker = mocks.mockObjectMakr();
+
+            // Act
+            var madeObject = objectMaker.make(mocks.mockClassExample, {name: "John", weight: 213, age: 4});
```

Comment:
Nit: for complex objects such as your `name`/`weight`/`age` literall, split across lines for readability.